<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>frmSFSAFT</class>
 <widget class="QWidget" name="frmSFSAFT" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Synthetic Focusing Technique</string>
  </property>
  <widget class="QPushButton" name="botOpen" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botCancel" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botClose" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFilePath" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>10</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>File []</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>220</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Processing Angle: </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progBar" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" >
     <number>24</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="botProc" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>0/0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>File Parameters</string>
   </property>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>A/D sampling rate (MHz)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>80</y>
      <width>96</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Time Offset (usec)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>No. Samples/Rec</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>140</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Element Space (mm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>20</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>No. Channels</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTimeOff" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask" >
     <string>###.99;0</string>
    </property>
    <property name="maxLength" >
     <number>6</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbSample" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>110</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>140</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Sound Velocity (m/s)</string>
    </property>
   </widget>
   <widget class="QwtCounter" name="cNChan" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="numButtons" >
     <number>2</number>
    </property>
    <property name="basicstep" stdset="0" >
     <double>1</double>
    </property>
    <property name="minValue" >
     <double>1</double>
    </property>
    <property name="maxValue" >
     <double>192</double>
    </property>
    <property name="value" >
     <double>1</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtElSpace" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask" >
     <string>###.99;0</string>
    </property>
    <property name="maxLength" >
     <number>6</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAD" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask" >
     <string>###.99;0</string>
    </property>
    <property name="maxLength" >
     <number>6</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSoundV" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>140</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask" >
     <string>9999; </string>
    </property>
    <property name="maxLength" >
     <number>4</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="4" margin="8" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
